free5gc-operator/config/crd/bases/workload.nephio.org_upfdeployments.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
Why are we downgrading the controller-gen version?

Reply:
fixed. v0.9.2 contoller-gen was in Makefile.